spec.html

```diff
@@ -28965,12 +28965,18 @@ <h1>
         <h1>
           HostEnsureCanCompileStrings (
             _calleeRealm_: a Realm Record,
+            optional _bodyText_: a String,
+            optional _direct_: a Boolean,
           ): either a normal completion containing ~unused~ or a throw completion
         </h1>
         <dl class="header">
           <dt>description</dt>
           <dd>It allows host environments to block certain ECMAScript functions which allow developers to interpret and evaluate strings as ECMAScript code.</dd>
         </dl>
+        <p>
+          _direct_ signifies whether the evaluation is a direct eval.
+          If _bodyText_ or _direct_ are present, the host environment may base its decision on their values.
```

Comment:
Without a specific prohibition, this is already true. Is it necessary to state this explicitly?

Reply:
Fair enough.